### PR TITLE
Cleaned up the Onboarding email-signup screen CSS

### LINKED ISSUE #7644: Clean up the Onboarding email-signup screen CSS
As of now, parts of the UI of the sign up flow are disorganized and create a less-than-ideal user experience.

Please make UI modifications to the current screen so that the design of the screen looks consistent with that documented in this Figma frame, a screenshot of which is also reproduced below:
https://www.figma.com/file/kiaTbWxdPsyqz6eeaT4M8l/Open-Library---Sign-In-Flow?node-id=79%3A2&t=gm520FiKGEaKFNWJ-1

Phase 7

This issue belongs to the epic https://github.com/internetarchive/openlibrary/issues/7694

### PR DESCRIPTION
**Pull request by imlakshay08**
Closes #7644

This PR updates the UI of signup form as mentioned in the screenshot of the issue. Though some minor changes are still required but achieved maximum of the given task.  

### Screenshot
Screenshot

### CODE REVIEW

File: openlibrary/templates/account/create.html [hunk 1]
```diff
@@ -43,37 +43,42 @@ <h1>$_("Sign Up")</h1>
...
         $if form.has_recaptcha:
-            <div class="formElement">
-                <div class="label smaller lighter">$_("If you have security settings or privacy blockers installed, please disable them to see the reCAPTCHA.")</div>
-                <div class="g-recaptcha" data-sitekey="$form['recaptcha'].public_key"></div>
-                <div class="input">
-                    $if form['recaptcha'].error:
-                        <span class="invalid clearfix">$_('Incorrect. Please try again.')</span>
-                </div>
+        <div class="formElement">
+            <div class="label smaller lighter">$_("Please disbale any privacy blockers to verify your identity.")</div>
```
Comment (cclauss) [inline openlibrary/templates/account/create.html:56]:
```suggestion
            <div class="label smaller lighter">$_("Please disable any privacy blockers to verify your identity.")</div>
```
```
codespell................................................................Failed
- hook id: codespell
- exit code: 65

openlibrary/templates/account/create.html:56: disbale ==> disable
```

File: openlibrary/templates/account/create.html [hunk 2]
```diff
@@ -43,36 +43,41 @@ <h1>$_("Sign Up")</h1>
         $if form.note:
             <div class="note">$form.note</div>
 
-        $def screenname_url(): $_('Your URL'): https://openlibrary.org/people/<span id="userUrl">$(form.username.value or _('screenname'))</span>
+        $def screenname_url(): $_('')
```
Comment (jimchamp) [inline openlibrary/templates/account/create.html:46]:
```suggestion
```
This is no longer needed.  Please remove `suffix` from the `field` call on line 49, below.  Also, remove `suffix` from the `field` definition, and the `span` that displays `suffix`.

File: openlibrary/plugins/upstream/forms.py [hunk 3]
```diff
@@ -75,7 +75,7 @@ class RegisterForm(Form):
     INPUTS = [
         Textbox(
             'email',
-            description=_('Your email address'),
+            description=_('Email Address *'),
```
Comment (jimchamp) [inline openlibrary/plugins/upstream/forms.py:78]: I'm not sure if this asterisk should always be displayed.  I think that it's meant to indicate that validation has failed.
Have raised these questions [here](https://github.com/internetarchive/openlibrary/issues/7644#issuecomment-1624178140).

File: openlibrary/plugins/upstream/forms.py [hunk 4]
```diff
@@ -113,7 +111,13 @@ class RegisterForm(Form):
         Checkbox(
             'ia_newsletter',
             description=_(
-                """I want to receive news, announcements, and resources from the <a href="https://archive.org/">Internet Archive</a>, the non-profit that runs Open Library."""
+                """Best of the archive : useful resources, unique stories, and fun find from our collections."""
+            ),
+        ),
+        Checkbox(
+            'ib_newsletter',
```
Comment (jimchamp) [inline openlibrary/plugins/upstream/forms.py:118]:
```suggestion
            'ia_events_notifications',
```
Not sure what `ib` stands for.  Let's call this `ia_events_notifications`.

File: openlibrary/templates/account/create.html [hunk 5]
```diff
@@ -43,36 +43,41 @@ <h1>$_("Sign Up")</h1>
...
 
+        <br>
+        <b>Subscriptions</b>
+        <br>
+        <label>
+            $:form.ia_newsletter.render() $:form.ia_newsletter.description
+        </label>
+        <br>
+        <label>
+            $:form.ib_newsletter.render() $:form.ib_newsletter.description
```
Comment (jimchamp) [inline openlibrary/templates/account/create.html:72]:
```suggestion
            $:form.ia_events_notifications.render() $:form.ia_events_notifications.description
```
Use updated name here.

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @imlakshay08, and sorry for the delay!  There are some open questions about validation that need to be addressed before this can be merged.

In the meantime, I've provided some suggested changes to improve the code.  In addition to these changes, we also want to honor a patron's notification preferences.  Previously, when the IA newsletter checkbox was checked, the patron would be signed up to both the newsletter and the event notifications.  Now, they should only be signed up for what was checked.

To do this, the account creation POST [handler](https://github.com/internetarchive/openlibrary/blob/c43bda22205d4d1add67a1f98320e3d51f4cf8f7/openlibrary/plugins/upstream/account.py#L268) must be modified.  If the `ia_newletter` checkbox is checked, `ml_best_of` should be added to the `notifications` list, and `ml_updates` should be added to `notifications` if `ia_events_notifications` is checked.

**Comment by imlakshay08** [comment]
> Thanks @imlakshay08, and sorry for the delay! There are some open questions about validation that need to be addressed before this can be merged.
> 
> In the meantime, I've provided some suggested changes to improve the code. In addition to these changes, we also want to honor a patron's notification preferences. Previously, when the IA newsletter checkbox was checked, the patron would be signed up to both the newsletter and the event notifications. Now, they should only be signed up for what was checked.
> 
> To do this, the account creation POST [handler](https://github.com/internetarchive/openlibrary/blob/c43bda22205d4d1add67a1f98320e3d51f4cf8f7/openlibrary/plugins/upstream/account.py#L268) must be modified. If the `ia_newletter` checkbox is checked, `ml_best_of` should be added to the `notifications` list, and `ml_updates` should be added to `notifications` if `ia_events_notifications` is checked.

Hey @jimchamp I have updated the pull request according to changes requested. Could you please re-review it?

File: openlibrary/plugins/upstream/account.py [hunk 6]
```diff
@@ -263,39 +263,44 @@ def is_plugin_enabled(self, name):
             or "openlibrary.plugins." + name in delegate.get_plugins()
         )
 
-    def POST(self):
```
Comment (cclauss) [inline openlibrary/plugins/upstream/account.py:266]: Indentation is significant in Python.  In the current code (red), this is a method under the class `account_create` under the definition that starts on line 234.  In the proposed code (green), it is a normal function and `self` will not be defined.

Comment (imlakshay08) [reply openlibrary/plugins/upstream/account.py:266]: Thank you for the review. I have updated it. Could you please re-review it?

File: openlibrary/plugins/upstream/forms.py (hunk 3 above)
Comment (jimchamp) [reply openlibrary/plugins/upstream/forms.py:78]: This will not be resolved until the asterisks and validation questions are answered.

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @imlakshay08!
I've unresolved two comments that you've marked as "resolved."  The `div` beneath the `if` statement in `create.html` needs to be indented by four spaces, and the `div`'s children must also be indented.
We'll also need some feedback about the asterisks and validation before this can be merged.

Can you fix the indentation issue, then rebase your code on `master`?  Instructions for rebasing are [here](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-updates-to-your-pull-request).  This is needed due to a merge conflict in `forms.py`.

**Comment by jimchamp** [comment]
@imlakshay08 are you still working on this?

**Comment by jimchamp** [comment]
This PR appears to be abandoned.  @imlakshay08, if you are still working on this, let me know by 4 September.

**Pull request closed without merging**